### PR TITLE
Decomp of func_8014A52C and WSL script

### PR DESCRIPTION
**Pull request by sonicspiral**
Decomp of `func_8014A52C` from `code_e0b30_len_b80`.
* updates to undefined_syms
* updates to variables.h
* added helper window_terminal.bat for windows users

Original mips to c output:
```
s16 func_8014A52C(void) {
    *(void *)0x80159AE2 = (u16)8;
    *(void *)0x80159AE4 = (u16)8;
    return (u16)8;
}
```

### CODE REVIEW

File: src/code_e0b30_len_b80.c [hunk 1]
```diff
@@ -4,7 +4,10 @@ INCLUDE_ASM("code_e0b30_len_b80", get_default_variation_for_song);
 
 INCLUDE_ASM("code_e0b30_len_b80", func_8014A498);
 
-INCLUDE_ASM("code_e0b30_len_b80", func_8014A52C);
+void func_8014A52C(void) {
+    D_80159AE2 = (s16)0x08;
```
Comment (ethteck) [inline src/code_e0b30_len_b80.c:8]: These casts shouldn't be needed since these variables are already s16s.

File: tools/windows_terminal.bat [hunk 2]
```diff
@@ -0,0 +1,6 @@
+:: replace with your distro
```
Comment (ethteck) [inline tools/windows_terminal.bat:1]: Could you explain what this does exactly? It sets up a few tabs in Windows Terminal? Seems kinda useful, but maybe it should take the distro as a flag or use the main distro by default.

**Comment by sonicspiral** [comment]
Thanks for the feedback! I'll start looking at the comments.

File: src/code_e0b30_len_b80.c (hunk 1 above)
Comment (sonicspiral) [reply src/code_e0b30_len_b80.c]: Sounds good, made the change under [this commit](https://github.com/ethteck/papermario/pull/45/commits/dedd132adcdb5683dbff926209d3e87d36e87354).

File: tools/windows_terminal.bat (hunk 2 above)
Comment (sonicspiral) [reply tools/windows_terminal.bat:1]: That's correct, it sets up tabs in the working directory that one could use mips_to_c or diff.py in. I double click on the .bat file and it opens the tabs for me. I like that idea, I'll look into setting the default distro automatically.

Comment (ethteck) [reply tools/windows_terminal.bat:1]: awesome! thanks

**Comment by sonicspiral** [comment]
@ethteck I'll propose the terminal with the default distro in another PR., wasn't able to find a quick solution to it. Feel free to merge  when ready 👍.

**Comment by ethteck** [comment]
sounds good! kk

**Pull request merged**